bindings/wasm/crate/src/lib.rs

```diff
@@ -90,7 +95,52 @@ pub fn create_inbound_session(
         .set(receiving_instance);
 
     match result {
-        Ok((_, plaintext)) => Ok(plaintext),
+        Ok((session_id, ciphertext_json)) => Ok(vec![
+            JsValue::from_str(&session_id),
+            JsValue::from_str(&ciphertext_json),
+        ]
+        .into_boxed_slice()),
+        Err(e) => Err(JsValue::from_str(&e.to_string())),
+    }
+}
+
+#[wasm_bindgen]
+pub fn encrypt_message(
+    sending_handle: &str,
+    session_id: &str,
+    message: &str,
+) -> Result<String, JsValue> {
+    let instances = INSTANCE_MAP.lock().unwrap();
+    let mut instance = instances
+        .get(sending_handle)
+        .ok_or("sending_handle not found")?
+        .take();
+
+    let result = instance.encrypt_message_serialized(session_id, message);
+    // Do we actually need to do this?
+    instances.get(sending_handle).unwrap().set(instance);
```

Comment:
`.unwrap` will panic on error. Becareful with these outside of prototyping code.